src/App/App.jsx

```
@@ -34,15 +34,21 @@ import useAuthentication from './useAuthentication'
 import useIsMounted from '../library/useIsMounted'
 import { getProjectIdFromLocation } from '../library/getProjectIdFromLocation'
 import { routes } from './routes'
+import { getIsAppADeployedPullRequestBranch } from '../library/getIsAppADeployedPullRequestBranch'
+import { getPullRequestRedirectUrl } from '../library/getPullRequestRedirectUrl'
```

ESLint

[error] 'getPullRequestRedirectUrl' is defined but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)